core/src/main/java/com/google/errorprone/bugpatterns/PreconditionsCheckNotNullPrimitive.java

```diff
@@ -157,10 +157,10 @@ private Fix createCheckArgumentOrStateCall(
 
     // Was the original call to Preconditions.checkNotNull a static import or not?
     if (methodInvocationTree.getMethodSelect().getKind() == Kind.IDENTIFIER) {
-      replacement.append(replacementMethod + "(");
+      replacement.append(replacementMethod).append("(");
```

Comment:
@ronshapiro Am I right to understand that you're suggesting something like this?

```java
if (methodInvocationTree.getMethodSelect().getKind() == Kind.IDENTIFIER) {
  fix.addStaticImport("com.google.common.base.Preconditions." + replacementMethod);
} else {
  replacement.append("Preconditions.")
}
replacement..append(replacementMethod).append('(');
```